gravitee-apim-console-webui/src/management/settings/api-portal-header/migrated/api-portal-header.component.ts

```diff
@@ -0,0 +1,182 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, OnDestroy, OnInit } from "@angular/core";
+import { EMPTY, Subject } from "rxjs";
+import { catchError, filter, switchMap, takeUntil, tap } from "rxjs/operators";
+import {
+  GioConfirmDialogComponent,
+  GioConfirmDialogData
+} from "@gravitee/ui-particles-angular";
+import { MatDialog } from "@angular/material/dialog";
+
+import { DialogData, HeaderDialogComponent } from "./header-dialog/header-dialog.component";
+
+import { EnvironmentApiHeadersService } from "../../../../services-ngx/environment-api-headers.service";
+import { ApiPortalHeader } from "../../../../entities/apiPortalHeader";
+import { SnackBarService } from "../../../../services-ngx/snack-bar.service";
+
+
+@Component({
+  selector: "app-api-portal-header",
+  templateUrl: "./api-portal-header.component.html",
+  styleUrls: ["./api-portal-header.component.scss"],
+  host: {
+    class: "bootstrap gv-sub-content"
+  }
+})
+export class ApiPortalHeaderComponent implements OnInit, OnDestroy {
+  // toDo: move documentation with banner particulars
+  private unsubscribe$: Subject<boolean> = new Subject<boolean>();
+  public settings = [];
+  public headersList: ApiPortalHeader[] = [];
+
+  displayedColumns: string[] = ["name", "value", "actions"];
+
+  constructor(
+    public environmentApiHeadersService: EnvironmentApiHeadersService,
+    private matDialog: MatDialog,
+    private snackBarService: SnackBarService,
+  ) {
+  }
+
+  ngOnInit(): void {
+    this.initListSubscription();
+    this.getList();
+  }
+
+  ngOnDestroy() {
+    this.unsubscribe$.next(true);
+    this.unsubscribe$.unsubscribe();
+  }
+
+  private initListSubscription() {
+    this.environmentApiHeadersService
+      .getHeadersList$()
+      .pipe(
+        catchError(_ => {
+          this.snackBarService.error('Error occurred.');
+          return [];
+        }),
+        takeUntil(this.unsubscribe$)
+      )
+      .subscribe((list: ApiPortalHeader[]): void => {
+        this.headersList = list;
+      });
+  }
+
+  private getList(): void {
+    this.environmentApiHeadersService
+      .getApiHeaders()
+      .pipe(takeUntil(this.unsubscribe$))
+      .subscribe();
+  }
+
+  public moveUp(apiPortalHeader: ApiPortalHeader): void {
+    const updatedHeader = { ...apiPortalHeader, order: apiPortalHeader.order - 1 };
+    this.environmentApiHeadersService
+      .updateApiHeader(updatedHeader)
+      .pipe(takeUntil(this.unsubscribe$))
+      .subscribe();
+  }
+
+  public moveDown(apiPortalHeader: ApiPortalHeader): void {
+    const updatedHeader = { ...apiPortalHeader, order: apiPortalHeader.order + 1 };
+    this.environmentApiHeadersService
+      .updateApiHeader(updatedHeader)
+      .pipe(takeUntil(this.unsubscribe$))
+      .subscribe();
+  }
+
+  public createHeader(): void {
+    this.matDialog.open(HeaderDialogComponent, {
+      width: "250px",
```

Comment:
WDYT to use one of this dialog size : https://particles.gravitee.io/?path=/docs/components-dialog-readme--docs 
The UI / UX team would prefer it to have a standard size ;)

Reply:
changed